ludwig/schema/features/image_feature.py

```diff
@@ -25,6 +27,8 @@ class ImageInputFeatureConfigMixin(BaseMarshmallowConfig):
         default="stacked_cnn",
     )
 
+    augmentation: BaseAugmentationConfig = AugmentationContainerDataclassField(default=[])
```

Comment:
Let's switch `BaseAugmentationConfig` to `List[BaseAugmentationConfig]`.

Reply:
doen